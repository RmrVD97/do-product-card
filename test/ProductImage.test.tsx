import React from 'react'
import renderer from 'react-test-renderer'
import { ProductCard } from '../src/components/ProductCard';
import {  product2 } from './products';
import { ProductImage } from '../src/components/ProductImage';





describe('ProductImage', () => {
    
    test('debe de mostrar el componente correctamente con el titulo personalizado', () => {

        const wrapper = renderer.create(
            <ProductImage img={product2.img} />
        )
        
        expect( wrapper.toJSON() ).toMatchSnapshot();
        
    }) ;

    test('debe de mostrar el componente correctamente', () => {

        const wrapper = renderer.create(
            <ProductCard product={product2 }>

             {   () => (<ProductImage />)
}
            </ProductCard>
        )
        
        expect( wrapper.toJSON() ).toMatchSnapshot();
        
    }) ;




}

)