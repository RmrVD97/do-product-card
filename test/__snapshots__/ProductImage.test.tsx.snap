// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage debe de mostrar el componente correctamente 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`ProductImage debe de mostrar el componente correctamente con el titulo personalizado 1`] = `
<img
  alt="Product"
  className="productImg undefined"
  src="./coffee-mug.png"
/>
`;
